Functions/Initialization/xASL_init_Iteration.m

```diff
@@ -87,38 +89,37 @@
         ModName = moduleName;
     end
     
-    % DARTEL & LongReg settings
-    if  strcmp(ModName,'DARTEL') || strcmp(ModName,'LongReg')
-        dbSettings.x.dir.LockDir = [dbSettings.x.dir.LockDir '_' x.P.STRUCT ];
```

Comment:
The lock-dir for LongReg is now different. And for DARTEL, no lockdir is now defined. Is that correct?

Reply:
line 41 sets a general lockdir that is used for all non-specific cases; LongReg specifically needed a different lock folder, as this crashed for Maurice

Reply:
Yes, something like that. I tested it after I rewrote this; it should be fine:) One of the two expected a session-extension in the lockdir, which wasn't there, which made it crash for Maurice. Then I think I did a small improvement: DARTEL should have no extension, whereas LongReg should have a subject extension.